src/utils/index.js

```diff
@@ -728,12 +728,18 @@ const FormioUtils = {
     lang,
   }) {
     // Get the prefix and suffix from the localized string.
-    const regex = `(.*)?100${decimalSeparator === '.' ? '\\.' : decimalSeparator}0{${decimalLimit}}(.*)?`;
+    let regex = '(.*)?100';
+    if (decimalLimit) {
+      regex += (decimalSeparator === '.') ? '\\.' : decimalSeparator;
+      regex += '0{' + decimalLimit + '}';
+    }
+    regex += '(.*)?';
     const parts = (100).toLocaleString(lang, {
       style: 'currency',
       currency,
       useGrouping: true,
-      maximumFractionDigits: decimalLimit
+      maximumFractionDigits: decimalLimit,
+      minimumFractionDigits: decimalLimit
```

Comment:
Are we sure we want to add minimumFractionDigits? Won't that require the full decimalLimit which is technically a "limit"?

Reply:
Looks like some tests aren't passing. As long as they pass I'm fine with this.

Reply:
Yeah getting tests to pass and will commit this. Thanks.